### DISCUSSION TITLE
Specified part does not exist in the package

### DISCUSSION THREAD

**Discussion by Griffin-Hartz** [ClosedXML/ClosedXML#2480] [Q&A]
I have no idea why I am getting this ExceptionUnhandled error: System.InvalidOperationException: 'Specified part does not exist in the package.'

I am just calling 
using (XLWorkbook workBook = new XLWorkbook(filePath))
with filepath as 
"C:\\Users\\Griffin.Hartz\\Downloads\\export (1).xlsx"

The package is definitely installed/exists, so I have no idea why visual studio would be throwing this error?

**Comment by vbjay** [comment]
Package is not nuget. Package is open XML terminology https://learn.microsoft.com/en-us/dotnet/api/documentformat.openxml.packaging.openxmlpackage?view=openxml-2.20.0 

Using your callstack to read the stack and understand what is throwing the exception would help you.
